code_of_conduct.md

```diff
@@ -0,0 +1,253 @@
+- At the discretion of the Standards Board and VP of Standards, instead of a face-to-face hearing, participants in a hearing may be allowed to participate telephonically or submit written information.
+- The complainant may choose to remain anonymous to the accused and the Standards Board and VP of Standards shall respect the complainant&#39;s wish.
+- The Standards Board and VP of Standards may conduct a formal hearing according to the following guidelines:
+  - The presiding officer, the VP of Standards, shall call the hearing to order.
+  - Admission of any person to the hearing shall be at the discretion of the presiding officer.
+  - The presiding officer shall expel from the hearing any person who, in his opinion, does not maintain proper decorum.
+  - Hearings normally shall be conducted in private. At the request of the accused, and subject to the discretion of the presiding officer other persons may be allowed to attend the hearing.
+  - In hearings involving more than one accused member, the presiding officer, in his discretion, may permit the hearings concerning each Member to be conducted separately.
+  - The complainant and the accused have the right to be assisted by any adviser they choose, at their own expense. The complainant and/or the accused are responsible for presenting his or her own case and, therefore, advisers are not permitted to speak or to participate directly in any hearing before the Standards Board and VP of Standards unless expressly asked to do so by the presiding officer. The complainant and the accused may present their case on separate dates in order to ensure anonymity of the complainant.
+  - The complainant, the accused, and the representative of the fact-finding team shall each have the right to make an open argument before the evidence is presented and a closing argument after all the evidence has been presented. The complainant and the accused may present their arguments on separate dates to ensure the anonymity of the complainant.
+  - The complainant, the accused, and the representative of the fact-finding team shall have the privilege of presenting witnesses, subject to the right of cross-examination by the Standards Board and VP of Standards. The complainant and the accused may present  their witnesses on separate dates to ensure the anonymity of the complainant.
+  - The Standards Board may accept pertinent records, exhibits and written statements as evidence for consideration at the discretion of the presiding officer.
+  - The presiding officer should remind those in attendance that the proceedings are an educational exercise and will not be perceived as a formal legal hearing.
```

Comment:
I agree that it should be mentioned that it shouldn't be perceived as a formal legal hearing, but calling it an "educational exercise" is kind of strange to me